libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestRequest+Internal.h

```diff
@@ -26,7 +26,9 @@
 
 @interface SFRestRequest ()
 
+@property (nonnull, nonatomic, strong, readwrite) NSMutableURLRequest *request;
 @property (nonatomic, copy) NSInputStream *(^requestBodyStreamBlock)(void);
+@property (nonatomic, copy) NSDictionary *requestBodyAsDictionary;
```

Comment:
Batch / composite requests need a json version of the request body. Either we compute it from the actual request body or if we happen to have it as construction time, we can save it for that purpose. It's the latter approach we used on android.